src/Features/CSharp/Portable/CodeFixes/AddImport/CSharpAddImportCodeFixProvider.cs

```diff
@@ -22,125 +22,142 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.AddImport
 {
-    using SymbolReference = ValueTuple<INamespaceOrTypeSymbol, MetadataReference>;
+    using Packaging;
+    using static AddImportDiagnosticIds;
 
-    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.AddUsingOrImport), Shared]
-    internal class CSharpAddImportCodeFixProvider : AbstractAddImportCodeFixProvider<SimpleNameSyntax>
+    internal static class AddImportDiagnosticIds
     {
         /// <summary>
         /// name does not exist in context
         /// </summary>
-        private const string CS0103 = "CS0103";
+        public const string CS0103 = "CS0103";
```

Comment:
Oh right.  THey're used in AddImport because we customize our behavior in it based on the ID.  For example, we have specialized logic that tries to map from the error span to the node.  It does things differently based on the error id.